package.json

```diff
@@ -5,7 +5,9 @@
     "start": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy"
+    "deploy": "docusaurus deploy",
+    "serve": "serve build/",
+    "build:serve": "docusaurus build && serve build/"
```

Comment:
Doesn't the start command serve the wiki?

Reply:
Yeah, it serves the wiki but the search can only work on a built site. As I wanted to style the results, I thought it'd be good to add a dev dependency for `serve` so to avoid having to deploy to vercel to check changes. Also I have been looking at the Playbooks page and wanted to try some stuff with the admin UI, which without serve, you can't use in development. It makes sense to be able to serve the build locally?

It's in dev deps so it shouldn't affect the actual build size?

Reply:
> Works better now :)
> I think that's fine, don't need to waste too much time to specific issues as the clear button, especially if this is a plugin and you don't really control the code.

Thanks for re-checking @pakokrew.  Reckon we're good to merge?